compiler/transforms/dispatch_regions.py

```diff
@@ -66,8 +67,20 @@ def dispatcher(block: Block, core_cond: builtin.i1, dispatch_rule: Callable):
         # in dominator block if changes made
 
         # FIXME: currently assuming that DM core is nb_cores - 1 and compute @ index 0
+
+        func_call = func.Call("snax_cluster_core_idx", [], [builtin.i32])
+
+        if self.pin_to_constants:
```

Comment:
I would remove the `pin_to_constants` flag alltogether, as it is a little bit confusing - why would we want it to be false?
If we want to debug, we can just disable the `pin-to-functions` pass? Then it is just a harmless unused attribute.
If you still want to keep it, maybe just let it default to true.

Reply:
Okay actually I agree! Thanks for the feedback Joren!